netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1332,6 +1376,7 @@ static void
 _net_ebpf_extension_sock_addr_copy_wfp_connection_fields(
     _In_ const FWPS_INCOMING_VALUES* incoming_fixed_values,
     _In_ const FWPS_INCOMING_METADATA_VALUES* incoming_metadata_values,
+    _In_opt_ const FWPS_CONNECT_REQUEST* connect_request,
```

Comment:
I did not see any invocation where this parameter is not null. Do you want to remove this and simplify the code?

Reply:
removed.